controllers/index.go

```diff
@@ -1,13 +1,24 @@
 package controllers
 
 import (
+	"github.com/foolin/goview/supports/ginview"
 	"github.com/gin-gonic/gin"
+	"github.com/gutakk/go-google-scraper/config"
+	"github.com/gutakk/go-google-scraper/middlewares"
 )
 
 func CombineRoutes(engine *gin.Engine) {
-	homeController := &HomeController{}
-	homeController.applyRoutes(engine)
+	new(HomeController).applyRoutes(engine)
```

Comment:
There should also be another group and middleware for `EnsureAuthenticationGroup`, right? This is needed to redirect to the login screen if a user is not authenticated.

Reply:
I design to let user able to access home page without authenticated but yes we should have that group when user try to access keyword page but I think we should done that in keyword related PR.
What do you think?

Reply:
It depends what will be on the home screen 😁 I assumed it would be the list of keywords/dashboard. So I leave that to you to decide if you plan to add this later.